src/app/globals.css

```
@@ -5,6 +5,7 @@
 @import '../styles/tabs.css';
 @import '../styles/spacing.css';
 @import '../styles/z-index.css';
+@import '../styles/animation.css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)